libminifi/src/sitetosite/RawSocketProtocol.cpp

```diff
@@ -394,98 +394,12 @@ bool RawSiteToSiteClient::getPeerList(std::vector<PeerStatus> &peers) {
     return false;
   }
 }
-
-int RawSiteToSiteClient::writeRequestType(RequestType type) {
```

Comment:
^ Same as comments above, This can stay and we can remove the duplicate in the base.

Reply:
Done, it seems github failed to handle the diff visualisaiton properly.

Reply:
There probably don't need to be any changes in this file, right? Seems that there have been.